### DISCUSSION TITLE
采用APISIX替换Nginx后项目部署端口监听问题

### DISCUSSION THREAD

**Discussion by li98936** [apache/apisix#5885] [Q&A]
项目采用的微前端方式进行部署，所有请求都是http://ip:9999/path 这样一个形式发起的，之前是采用Nginx对9999端口进行监听转发，这个在APISIX中需要怎么处理呢？目前我们的APISIX只有请求到9080端口才能被监听到，这个是需要去修改前端请求地址吗？还是说可以不用修改前端代码能够通过APISIX处理？

**Comment by tokers** [comment]
https://github.com/apache/apisix/blob/master/conf/config-default.yaml#L24
